ic-agent/src/agent/agent_error.rs

```diff
@@ -165,25 +165,75 @@ impl Display for HttpErrorPayload {
     }
 }
 
-fn is_plain_text_utf8(content_type: &Option<String>) -> bool {
-    // text/plain is also sometimes returned by the replica (or ic-ref),
-    // depending on where in the stack the error happens.
+fn format_as_text(content_type: &Option<String>) -> bool {
```

Comment:
I would prefer to name this predicate as a true/false question; `is_text` or `can_format_as_text`.

This new name implies that it formats it.

Reply:
Good idea.  Thanks!